docs/namespace.md

```diff
@@ -7,10 +7,9 @@ you run Ark client commands.
 ## Edit the example files
 
 The Ark repository includes [a set of examples][0] that you can use to set up your Ark server. The
-examples place the server in the `heptio-ark-server` namespace, and backup/schedule/restore/config
-data in the `heptio-ark` namespace.
+examples place the server and backup/schedule/restore/config data in the `heptio-ark` namespace.
 
-To run the server in another namespace, you edit the relevant files, changing `heptio-ark-server` to
+To run the server in another namespace, you edit the relevant files, changing `heptio-ark` to
```

Comment:
One thing not explicitly called out here is that the cloud-credentials secret also needs to be created in the alternate namespace. It's templatized in each cloud-provider's walkthrough, but maybe worth just referencing it here too.

Reply:
Will do